tests/integration_tests/build/test_gdb.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A test that ensures that firecracker builds with GDB feature enabled at integration time."""
+
+import platform
+
+import pytest
+
+import host_tools.cargo_build as host  # pylint:disable=import-error
+
+MACHINE = platform.machine()
+TARGET = "{}-unknown-linux-musl".format(MACHINE)
+
+
+@pytest.mark.skipif(MACHINE != "x86_64", reason="GDB runs only on x86_64.")
```

Comment:
We can do:
```Rust
#[cfg(all(feature = "gdb", target_arch = "aarch64"))]
compile_error!("gdb support is not implemented for aarch64");
```
But I don't think we need a negative integration test for `aarch64` because we will add support later anyways.

Reply:
Added this now so we will get the compile error on arm